pages/create.js

```diff
@@ -7,17 +6,18 @@ export default function CreatePage() {
   const router = useRouter();
 
   async function addItem(data) {
+    console.log(data);
     try {
       await fetch("api/items", {
         method: "POST",
```

Comment:
i am not shure about this "POST" method because in your `api/items/[id.js]`i only see the GET PUT PATCH and DELETE method. maybe i don't get it and you can explain ?

Reply:
The POST handler of backend is in `api/items/index.js`, because I add new item to home page, not to another item :)
In `api/items/[id.js]` I have only methods, witch go anything with specific item.

Reply:
ok thanks :)